sdk/include/ddk/d3dkmthk.h

```diff
@@ -5649,11 +5649,13 @@ typedef _Check_return_ NTSTATUS (APIENTRY *PFND3DKMT_CANCELPRESENTS)(_In_ D3DKMT
 
 #endif
 
+#ifndef __REACTOS__
```

Comment:
Why does this need to be put if a (not) REACTOS block?

Reply:
This is a WSL thing,

Reply:
The question is whether these triggered build errors (in which case I would understand the need of the ifndef REACTOS block), or not.

Reply:
they just need to be wrapped ina  WDDM2.2 > check it seems or so. 

Looks like they just threw these in there, yes it breaks build